### ISSUE TITLE
Action potential related terms

### ISSUE THREAD

**Issue by jacakrj1** [jhuapl-bio/pathogenesis-gene-ontology#157]
Term PATHGO:0000023 'disruption of action potential propagation' has a comment that states "inconsistent as a subclass as this is a consequence of ion channel activity modulation; should be moved".  This part of the ontology needs to be looked at and revised.

**Comment by genegodbold** [comment]
@jproesch FWIW, I agree that these sorts of things should be dealt with through a "modulation of ion channel in other organism" term. If you don't specify "in other organism" all sorts of normal neurological function might get dumped in here.

**Comment by jproesch** [comment]
@genegodbold  agreed.  For now I will add specificity by adding host terms to labels and leave this open as a known issue.

**Comment by genegodbold** [comment]
@jproesch "Host" works only in cases where you want to exclude other things. But for animal venom components (snake/spider/Cnidarian/scorpion/what have you) the specification needs to be made for "in other organism". Because there are more venom components that affect receptors "in other organisms", particularly ion channels, than there are microbial virulence factors (I'm guessing).

**Comment by jproesch** [comment]
@genegodbold, yep, I caught myself on that when I made the changes and did indeed use "other organism"!
